extensions/adobe/experience/marketo/account-marketo.schema.json

```diff
@@ -0,0 +1,124 @@
+  ],
+  "$id": "https://ns.adobe.com/experience/marketo/account-marketo",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Account",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/account"],
+  "meta:extends": ["https://ns.adobe.com/xdm/context/account-crm"],
+  "description": "Marketo specific Account fields.",
+  "definitions": {
+    "account-marketo": {
+      "properties": {
+        "marketo:accountID": {
+          "title": "Marketo Account Identifier",
+          "description": "Marketo account identifier.",
+          "type": "string"
+        },
+        "marketo:instanceID": {
+          "title": "Instance Identifier",
+          "description": "Instance identifier.",
+          "type": "string"
+        },
+        "marketo:org": {
+          "properties": {
+            "marketo:notes": {
+              "title": "Account Notes",
+              "description": "Account notes.",
+              "type": "string"
+            },
+            "marketo:accountType": {
+              "title": "Account Type",
+              "description": "Account type.",
+              "type": "string"
+            },
+            "marketo:primaryInterestID": {
+              "title": "Primary Interest Id",
+              "description": "Primary interest id.",
+              "type": "string"
+            },
+            "marketo:private": {
+              "title": "Private Flag",
+              "description": "Private flag.",
+              "type": "boolean"
+            },
+            "marketo:additionalFields": {
+              "title": "Additional Fields",
+              "description": "Additional fields.",
+              "type": "string"
+            }
+          }
+        },
+        "marketo:namedAccount": {
+          "properties": {
+            "marketo:opportunityAmount": {
+              "title": "Opportunity Amount",
+              "description": "Opportunity amount.",
+              "type": "number"
+            },
+            "marketo:opportunityCount": {
+              "title": "Opportunity Count",
+              "description": "Opportunity count.",
+              "type": "number"
+            },
+            "marketo:percentiles": {
+              "title": "Percentile Measures",
+              "description": "Percentile measures.",
+              "type": "array",
+              "items": {
+                "type": "number"
+              }
+            },
+            "marketo:scores": {
+              "title": "Score Measures",
+              "description": "Score measures.",
+              "type": "array",
+              "items": {
+                "type": "number"
+              }
+            },
+            "marketo:membershipCount": {
+              "title": "Membership Count",
+              "description": "Membership count.",
+              "type": "number"
```

Comment:
Integer?  Min of 1?

Reply:
+1 - Can you ever have a decimal count (1.4 members). If not, then change this to integer with a min/max range if possible.

Reply:
Fixed. Will wait for MARKETO to provide range if any.